packages/salesforcedx-vscode-core/src/commands/forceLightningInterfaceCreate.ts

```diff
@@ -26,13 +26,14 @@ import {
   CompositeParametersGatherer,
   LightningFilePathExistsChecker,
   SelectFileName,
-  SelectStrictDirPath,
+  SelectOutputDir,
   SfdxCommandlet,
   SfdxCommandletExecutor,
   SfdxWorkspaceChecker
 } from './commands';
 
 const LIGHTNING_INT_EXTENSION = '.intf';
+const LIGHTNING_METADATA_DIR = 'aura';
```

Comment:
Bryan, I'm seeing a whole bunch of LIGHTNING_METADATA_DIR constant definitions across multiple files (among other LIGHTNING_*_EXTENSION constants). We should create a file for these metadata constants and store them in there especially if we're going to be handling other metadata types in a similar fashion.

Reply:
Yeah I very much agree. I'm going to create a refactor story for these template commands and as part of it I want to create a configuration file the commands can read. It'll contain things like file extension, metadata dir name, requires subdirectory. Since the refactoring has quite a few changes I wanted to keep it separate from this work.